lib/param_parsing/toml.go

```diff
@@ -28,15 +28,18 @@ func getParamsTOML(params Params) (Params, error) {
 
 		if viperParser.Get("bin") != nil {
 			params.CustomBinaryPath = viperParser.GetString("bin")
+			logger.Debugf("Using bin from toml file %q: %q", tomlPath, params.CustomBinaryPath)
 		}
 		if viperParser.Get("log-level") != nil {
 			params.LogLevel = viperParser.GetString("log-level")
```

Comment:
Any reason to not debug-log this config option?
```suggestion
			params.LogLevel = viperParser.GetString("log-level")
			logger.Debugf("Using \"log-level\" from %q: %q", tomlPath, params.LogLevel)
```

Reply:
I'm not sure - mainly that it doesn't contribute much to the actual installation process. But happy to

Reply:
I'd say it's best to keep all of them consistent: once others have a `logger.Debugf()`, others preferably should too. Thanks.

Reply:
Coo - it's done :D